.buildconfig.yml

```diff
@@ -160,6 +160,10 @@ projects:
     path: components/browser/awesomebar
     description: 'A customizable awesomebar component for browsers.'
     publish: true
+  browser-containers:
```

Comment:
My gut reaction here is that I think this should be a `feature` component? 🤔 Similar to `feature-top-sites` and `feature-tab-collections`.
And I could imagine that we may add more "container feature code" here?

Reply:
@pocmo Heh, it was originally a feature component but I suggested to gl that it should be a browser component. I figured it didn't have any feature aspect to it but could be used as part of our features instead as we decide how to build those. For example, PWA containers or the desktop version of multi-account containers. We do use feature components within other feature components though.

Either way could work really.

Reply:
@pocmo @jonalmeida I would like a stronger preference if anyone has one. I already changed it from `feature-contextual-identities` to `browser-container` once. Also, finalizing a naming would also be great. `feature-contextual-identities` ran into the issue with CI error: `RuntimeError: Treeherder group and symbol names must not be longer than 25 characters: feature-contextualidentities`. Alternatively, we could call this `feature-containers`.

Reply:
Just as an FYI, there's a workaround to make longer names work with treeherder (keep the long name and add a shorter override for treeherder):
https://github.com/mozilla-mobile/android-components/blob/11e259c2c347b7d608a3c9cc616d1549b9a7427c/taskcluster/ci/build/kind.yml#L96

> We do use feature components within other feature components though.

Yeah, dependencies on the same level are fine.

>  Alternatively, we could call this feature-containers

That sounds good to me! 👍